docs/types/value-types.rst

```diff
@@ -498,7 +498,7 @@ terminate the string literal. Newline only terminates the string literal if it i
 Hexadecimal Literals
 --------------------
 
-Hexadecimal literals are prefixed with the keyword ``hex`` and are enclosed in double or single-quotes (``hex"001122FF"``). Their content must be a hexadecimal string and their value will be the binary representation of those values.
+Hexadecimal literals are prefixed with the keyword ``hex`` and are enclosed in double or single-quotes (``hex"001122FF"``), and the can also be split into multiple consecutive parts (``hex"00112233" "44556677"`` is equivalent to ``hex"0011223344556677"``). Their content must be a hexadecimal string and their value will be the binary representation of those values.
```

Comment:
I would actually prefer 
```suggestion
Hexadecimal literals are prefixed with the keyword ``hex`` and are enclosed in double or single-quotes (``hex"001122FF"``), and the can also be split into multiple consecutive parts (``hex"00112233" hex"44556677"`` is equivalent to ``hex"0011223344556677"``). Their content must be a hexadecimal string and their value will be the binary representation of those values.
```

Reply:
@chriseth Do you mean that `hex"00112233" hex"44556677"` should be an allowed syntax? Are you suggesting that each part of a hex string should start with `hex`?

In the original issue description here https://github.com/ethereum/solidity/issues/7292#issue-483992621 the `hex"00112233" "44556677"` was given as an example, and from what I understood from the original issue description and from the previous comments on this PR, I have tried to implement the following:

* For regular strings, no parts should start with `hex`.

Right:
```
"aa" "bb" "cc"
"dd"
```
Wrong:
```
"aa" "bb" hex"cc"
"dd"
```

* For hex strings on a single line, only the first part should start with `hex` and the rest of the strings on the same line should not.

Right:
```
hex"aa" "22"
```
Wrong:
```
hex"aa" hex"22"
```

* For hex strings on multiple lines, only the first part on each line should start with `hex`.

Right:
```
hex"11" "22"
hex"33" "44"
```
Wrong:
```
hex"11" "22"
"33" "44"
```

What do you think of the above rules?

Reply:
@ghallak newlines are fragile creatures, so I would like to avoid having to distinguish newlines from other whitespace as much as possible, so I think it would be better to force people to use a `hex` prefix even for non-newline whitespace. What do the others think?

Reply:
I've adjusted the docs here 7a2b6dd0e75d164f2764c88fd8f0c1ee2a03c469